src/theme/createSaleorTheme/createSaleorTheme.tsx

```diff
@@ -26,8 +26,8 @@ export const createTheme = (colors: SaleorThemeColors): SaleorTheme =>
       ...overrides(colors, fontFamily),
       MuiCard: {
         root: {
-          borderRadius: 8,
-          borderWidth: 0,
+          border: `1px solid ${colors.main[6]}`,
```

Comment:
Card borderColor in darkmode is actually main[4] now (check figma). That's how I would do it
```suggestion
          border: "1px solid",
          borderColor: colors.theme === "dark" ?  colors.main[4] : colors.main[6],
```